### PR TITLE
Remove broken links

### PR DESCRIPTION
**Pull request by wrt54g**
## Proposed change

I removed 3 more broken links in the Help page.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/help/index.markdown [hunk 1]
```diff
@@ -34,8 +34,6 @@ Have you found an issue in your Home Assistant installation? Please report it. R
 - [Why we can't have the Internet of Nice Things: A home automation primer](https://www.openwest.org/custom/description.php?id=92) at [OpenWest 2017](https://www.openwest.org) - July 2017
 - [Home Automation with Home Assistant](https://github.com/jjmontesl/talk-hass-pydaygalicia2017) at [PyDay Galicia 2017](https://pyday2017.python-vigo.es/gl/) - June 2017
 - [Home Automation with Python](https://www.youtube.com/watch?v=KNFZSSCPUyM) at [GLT 2017](https://glt17.linuxtage.at) - April 2017
-- [Home Assistant workshop](https://github.com/home-assistant/home-assistant-assets/tree/master/german/2017-clt-workshop) at [CLT 2017](https://chemnitzer.linux-tage.de/2017/de/) - March 2017
```
Comment (fabaff) [inline source/help/index.markdown:37]: That link would be: https://github.com/home-assistant-ecosystem/workshops/tree/main/2017-clt-workshop

File: source/help/index.markdown [hunk 2]
```diff
@@ -34,8 +34,6 @@ Have you found an issue in your Home Assistant installation? Please report it. R
...
-- [Home Assistant - Erweiterungen (Platforms/Components)](https://github.com/home-assistant/home-assistant-assets/tree/master/german/2016-puzzle) at [Puzzle ITC](https://www.puzzle.ch/de/) - December 2016
```
Comment (fabaff) [inline source/help/index.markdown:38]: That link would be: https://github.com/home-assistant-ecosystem/workshops/tree/main/2016-puzzle

File: source/help/index.markdown [hunk 3]
```diff
@@ -77,7 +75,6 @@ Don't miss the regular [Home Assistant podcasts](https://hasspodcast.io/).
 - [Castálio Podcast - Episódio 102: Marcelo Mello - Red Hat e Automação Residencial com Home Assistant](https://youtu.be/hZq8ucpzjCs) - May 2017
 - [Paulus Schoutsen and Home Assistant - Episode 8](https://codepop.com/open-sourcecraft/episodes/paulus-schoutsen/) - March 2017
 - [Zammad, Home Assistant and Freifunk - are the winner of the Thomas-Krenn-Awards 2017](https://www.thomas-krenn.com/de/tkmag/allgemein/zammad-home-assistant-und-freifunk-das-sind-die-gewinner-des-thomas-krenn-awards-2017/) - March 2017
-- [10 open source software tools for developing IoT applications](https://www.cbronline.com/news/internet-of-things/10-open-source-software-tools-developing-iot-applications/) - March 2017
```
Comment (fabaff) [inline source/help/index.markdown:80]: We could use archive.org: http://web.archive.org/web/20200926222601/https://www.cbronline.com/news/internet-of-things/10-open-source-software-tools-developing-iot-applications/

I guess that we can remove that link as the section about Home Assistant is very short.

**Review by fabaff — CHANGES_REQUESTED** [review]

**Comment by wrt54g** [comment]
@fabaff I fixed them now.

File: source/help/index.markdown [hunk 4]
```diff
@@ -34,6 +34,8 @@ Have you found an issue in your Home Assistant installation? Please report it. R
 - [Why we can't have the Internet of Nice Things: A home automation primer](https://www.openwest.org/custom/description.php?id=92) at [OpenWest 2017](https://www.openwest.org) - July 2017
 - [Home Automation with Home Assistant](https://github.com/jjmontesl/talk-hass-pydaygalicia2017) at [PyDay Galicia 2017](https://pyday2017.python-vigo.es/gl/) - June 2017
 - [Home Automation with Python](https://www.youtube.com/watch?v=KNFZSSCPUyM) at [GLT 2017](https://glt17.linuxtage.at) - April 2017
+- - [Home Assistant workshop](https://github.com/home-assistant-ecosystem/workshops/tree/main/2017-clt-workshop) at [CLT 2017](https://chemnitzer.linux-tage.de/2017/de/) - March 2017
```
Comment (fabaff) [inline source/help/index.markdown:37]: `-` instead of `- -`.

see https://deploy-preview-22615--home-assistant-docs.netlify.app/help/

**Review by fabaff — CHANGES_REQUESTED** [review]

**Comment by fabaff** [comment]
Superseded by #22904

**Pull request closed without merging**